frontend/src/APIClients/mutations/TestMutations.ts

```diff
@@ -0,0 +1,18 @@
+import { gql } from "@apollo/client";
+
+const PUBLISH_TEST_MUTATION = gql`
+  mutation PublishTestMutation($testId: String!) {
+    publishTest(testId: $testId) {
+        id
+        name
+        admin
+        questions
+        grade
+        assessmentType
+        status
+        curriculumCountry
+        curriculumRegion
```

Comment:
do we need to return all these fields here?

Reply:
its in TestRequestDTO so i added it in